content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Container Cost Allocation
+kind: documentation
+disable_toc: false
+private: true
+further_reading:
+- link: "https://docs.datadoghq.com/cloud_cost_management/?tab=aws#overview"
+  tag: "Documentation"
+  text: "Cloud Cost Management"
+---
+
+{{< jqmath-vanilla >}}
+
+## Overview
+
+Datadog Cloud Cost Management (CCM) automatically allocates EC2 compute cost in Kubernetes and ECS clusters to individual pods and tasks running in those clusters. Use cost metrics enriched with tags from pods, nodes, containers, and tasks to visualize container workload cost in the context of your entire cloud bill.
+
+{{< img src="cloud_cost/container_cost_allocation/cost_allocation_table.png" alt="Cloud cost allocation table showing requests and idle costs over the past week" style="width:100%;" >}}
+
+## Prerequisites
+
+1. Set up and configure [AWS Cloud Cost integration][1].
+1. At least one of the following must be running:
+    - [**Datadog Agent**][2] in a Kubernetes environment using EC2 instances.
+    - **Datadog Container Monitoring** in containers running in ECS tasks. The container agent must be configured to use [orchestrator tag cardinality][3].
```

Comment:
orchestrator tag cardinality may not actually be necessary. @jeannei did you get confirmation on this?

Reply:
got confirmation from the #processes team that we can remove this req